pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -22,6 +22,14 @@ const (
 	// inbound listeners won't be generated in that case
 	KumaGatewayAnnotation = "kuma.io/gateway"
 
+	// KumaBuiltinGatewayAnnotation allows to mark a pod as serving a built-in
+	// Gateway with the name annotated here.
+	// This alters the dataplane generated by the pod controller
+	KumaBuiltinGatewayAnnotation = "kuma.io/builtin-gateway"
+
+	// KumaGatewayListenerAnnotation is a tag used to select Gateway listeners.
+	KumaGatewayListenerAnnotation = "gateways.kuma.io/listener"
```

Comment:
Since this is a well-known Kuma tag not a Kubernetes annotation, we should rename and move it somewhere different.